packages/utilities/psammead-storybook-helpers/src/input-provider.js

```diff
@@ -0,0 +1,19 @@
+import { boolean, text, select } from '@storybook/addon-knobs';
+import LANGUAGE_VARIANTS from './text-variants';
+
+const inputProvider = (names, componentFunction) => () => {
```

Comment:
Is there a way you can unit test this logic? Currently this component only has a unit test that asserts its a function